SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -8,15 +8,144 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let walletText = "Кошелек"
+        static let profileText = "Профиль"
+        static let currentBalanceText = "Текущий баланс:"
+        static let showSettings = "Показать настройки"
+        static let rub = "рублей"
+        static let refillTheBalanceButtonTitle = "Пополнить"
+        static let actionSheetRefillTheBalanceText = "Баланс пополнен"
+        static let actionSheetWriteOffFundsText = "Деньги списаны"
+        static let actionSheetInsufficientFunds = "Недостаточно средств"
+        static let emptyString = ""
+        static let openTwoCheck = "Второй счет открыт"
+        static let openCheckText = "Открыть счет"
+        static let refillTheBalanceAlertButton = "Пополнить на 50 рублей?"
+        static let transfersMoneyAlertTitle = "Переводы"
+        static let transfersMoneyAlertButton = "Перевести"
+        static let transferMoneyAlertTitle = "Перевести 50 рублей?"
+        
+        
+    }
+    
+    @State var isOnToogle = false
+    @State var balance = 0
+    @State var balanceTwo = 50
+    @State var isRefillTheBalanceAlert = false
+    @State var isBalanceToppedUp = false
+    @State var actionSheetText = Constants.emptyString
+    @State var isTransfersMoneyAlert = false
+    @State var isCreateСheck = false
+    @State var openTwoCheck = Constants.emptyString
+    
+    
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            ZStack {
+                HStack {
+                    VStack {
+                        Text(Constants.walletText)
+                        Spacer().frame(height: 20)
+                        transfersMoneyButton()
+                        Spacer().frame(height: 10)
+                        refillTheBalanceButton()
+                        Spacer().frame(height: 150)
+                        Text(Constants.profileText)
+                        Spacer().frame(height: 20)
+                        openCheckButton()
+                        Spacer().frame(height: 100)
+                    }
+                    Spacer()
+                }
+                Spacer()
+                RoundedRectangle(cornerRadius: 10).fill(Color.yellow).offset(x: isOnToogle ? 200: 0)
```

Comment:
```suggestion
                RoundedRectangle(cornerRadius: 10).fill(Color.yellow).offset(x: isOnToogle ? 200: 0)
```
вынеси плиз магические числа все